XCC.md

```diff
@@ -0,0 +1,27 @@
+# Cross contract call
```

Comment:
Let's remove this file, and I'll be writing a proper AIP.

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/560/commits/53c220eef327617c4caa8f871b0e633731f36124